nav2_collision_monitor/CMakeLists.txt

```diff
@@ -57,6 +58,7 @@ add_library(${monitor_library_name} SHARED
 add_library(${detector_library_name} SHARED
   src/collision_detector_node.cpp
   src/polygon.cpp
+  src/velocity_polygon.cpp
```

Comment:
It seems that VelocityPolygons are not going to be added to a CollisionDetector. Thus, it seems that source-file is unnecessary for CD, only used for CM.

Reply:
I think this is needed because velocity_polygon is included in polygon.cpp